MTA-6811, 6812, 6813, 6814, 6815, 6816 CQA April Developer Lightspeed work (#357)
migtools/mta-documentation: docs/topics/developer-lightspeed/ref_llm-provider-configurations.adoc

[role="_abstract"]
{mta-dl-full} is large language model (LLM) agnostic and integrates with an LLM of your choice. To enable {mta-dl-plugin} to access your large language model (LLM), you must enter the LLM provider configurations in the `provider-settings.yaml` file.

> The `provider-settings.yaml` file has a list of LLM providers that {mta-dl-plugin} supports by default. The mandatory environment variables are different for each LLM provider. Depending on the provider that you choose, you can configure additional environment variables for a model in the `provider-settings.yaml` file. You can also enter a new provider with the required environment variables, the base URL, and the model name.

Reviewer: "Contains" is a better word in this case. 
```suggestion
The `provider-settings.yaml` file contains a list of LLM providers that {mta-dl-plugin} supports by default. The mandatory environment variables are different for each LLM provider. Depending on the provider that you choose, you can configure additional environment variables for a model in the `provider-settings.yaml` file. You can also enter a new provider with the required environment variables, the base URL, and the model name.
```

[NOTE]
====

> You can use the AWS CLI to verify that you have command line access to AWS services before you proceed with the `provider-settings` configurations.

Reviewer: ```suggestion
To verify command-line access to Amazon Web Services (AWS) before configuring `provider-settings`, use the AWS CLI.
```
